src/AppEntry.js

```diff
@@ -13,7 +14,9 @@ const AppEntry = () => {
       }}
     >
       <Provider store={registry.getStore()}>
-        <App />
+        <Authentication>
```

Comment:
This should be wrapped in QueryClientProvider. Right now, if you open /insights/connector, you get
```
No QueryClient set, use QueryClientProvider to set one
```